app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/MessageGeo.java

```diff
@@ -0,0 +1,17 @@
+package com.teamagam.gimelgimel.app.model.ViewsModels;
+
+import com.teamagam.gimelgimel.app.model.entities.GeoTextSample;
+
+/**
+ * Created by Gil.Raytan on 10-Jul-16.
+ * A class for geo messages (using LocationEntity as its content)
```

Comment:
If the documentation doesnt add any extra knowledge gained by reading the class name/declaration, it has no use.
The documentation can only "rot" as time goes by and get out of sync with the code-base.

We did make a decision to write documentation to every class and public method, but times shows that this decision is of no use. 

We should document only when our class/method naming is incapable of delivering all that there is to know. 
But that's just my opinion.